docs/sdk/src/reference_docs/chain_spec_runtime/tests/chain_spec_builder_tests.rs

```diff
@@ -8,189 +9,184 @@ fn wasm_file_path() -> &'static str {
 
 const CHAIN_SPEC_BUILDER_PATH: &str = "../../../../../target/release/chain-spec-builder";
 
+macro_rules! bash(
+	( chain-spec-builder $($a:tt)* ) => {{
+		let path = get_chain_spec_builder_path();
+		spawn_with_output!(
+			$path $($a)*
+		)
+		.expect("a process running. qed")
+		.wait_with_output()
+		.expect("to get output. qed.")
+
+	}}
+);
+
 fn get_chain_spec_builder_path() -> &'static str {
-	// dev-dependencies do not build binary. So let's do the naive work-around here:
-	let _ = std::process::Command::new("cargo")
-		.arg("build")
-		.arg("--release")
-		.arg("-p")
-		.arg("staging-chain-spec-builder")
-		.arg("--bin")
-		.arg("chain-spec-builder")
-		.status()
-		.expect("Failed to execute command");
+	run_cmd!(
+		cargo build --release -p staging-chain-spec-builder --bin chain-spec-builder
+	)
+	.expect("Failed to execute command");
 	CHAIN_SPEC_BUILDER_PATH
 }
 
+#[docify::export_content]
+fn cmd_list_presets(path: &str) -> String {
+	bash!(
+		chain-spec-builder list-presets -r $path
```

Comment:
nit: would be great to name this `$runtime_path`, and in the rest of the tests too.